packages/store-sync/src/getRecords.bench.ts

```diff
@@ -0,0 +1,36 @@
+import { bench, describe } from "vitest";
+import { getComponentEntities, getComponentValue } from "@latticexyz/recs";
+import { eq } from "drizzle-orm";
+import { mudStoreTables } from "./sqlite";
+import { blocks } from "../test/blocks";
+import {
+  components,
+  db,
+  recsStorageAdapter,
+  sqliteStorageAdapter,
+  tables,
+  useStore,
+  zustandStorageAdapter,
+} from "../test/utils";
+
+for (const block of blocks) {
+  await recsStorageAdapter(block);
+  await zustandStorageAdapter(block);
+  await sqliteStorageAdapter(block);
+}
+
+describe("Get Records", () => {
+  bench("recs: `getComponentValue`", async () => {
+    for (const entity of getComponentEntities(components.NumberList)) {
+      getComponentValue(components.NumberList, entity);
+    }
+  });
+
+  bench("zustand: `getRecords`", async () => {
+    useStore.getState().getRecords(tables.NumberList);
+  });
+
+  bench("sqlite: `select`", async () => {
+    db.select().from(mudStoreTables).where(eq(mudStoreTables.name, "NumberList")).all();
+  });
+});
```

Comment:
If the benchmark results output are still nice, might be better to have e.g. RECS bench tests in `recsStorage.bench.ts` that does has a `getRecord` and `getRecords` benchmark test